ui/src/commonMain/kotlin/com/materiiapps/gloom/ui/components/LargeSegmentedButton.kt

```diff
@@ -26,12 +27,15 @@ import androidx.compose.ui.unit.dp
 fun RowScope.LargeSegmentedButton(
     icon: Any,
     iconDescription: String? = null,
-    text: String
+    text: String,
+    onClick: () -> Unit = {},
+    enabled: Boolean = true
 ) {
     Column(
         horizontalAlignment = Alignment.CenterHorizontally,
         verticalArrangement = Arrangement.spacedBy(12.dp, Alignment.CenterVertically),
         modifier = Modifier
+            .clickable(enabled = enabled, role = Role.Button) { onClick() }
```

Comment:
Role needs to be imported

Reply:
I am sorry, but I do not understand what this means. What do you mean by "Role needs to be imported"?

Reply:
`Role` is a class. [As mentioned in my suggestion](https://github.com/MateriiApps/Gloom/pull/33#discussion_r1382725299) that caused this build error, you should import the class.

Reply:
As a parameter to the segmented button, seems as it won't always be used as a button